<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlDialog</class>
 <widget class="QWidget" name="CtrlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font:16px;
    color: rgb(240,240,240);
	background-color: rgb(0, 0, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
}
QGroupBox
{
border:3px solid white
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>1</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>5</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>8</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>2</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>9</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>3</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>4</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>6</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>10</string>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>0</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>7</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBLinput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid white;
background:white;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTNOK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTNreturn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBLTips">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color:white;
font:16px ;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>密码错误 请重新输入</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
